openff/toolkit/typing/engines/smirnoff/forcefield.py

```diff
@@ -1124,6 +1124,7 @@ def to_file(self, filename, io_format=None, discard_cosmetic_attributes=False):
 
     # TODO: Should we also accept a Molecule as an alternative to a Topology?
     @requires_package("openmm")
+    @requires_package("openff.interchange")
```

Comment:
I'm pretty sure we handle this at the packaging level - are you installing everything from source? Did you end up in a state where the toolkit was installed but Interchange was not?

Reply:
No, I only included that because the function right below, create_interchange() has that decorator

Reply:
That decorator might not be needed then